ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -35,9 +35,9 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: openshift-qe-ocp-qe-perfscale-ci-tests
+    workflow: openshift-qe-control-plane
   timeout: 5h0m0s
-- as: ocp-qe-perfscale-aws-ci-tests-small-scale-kube-burner
+- as: ocp-qe-perfscale-aws-cp-small-scale-kube-burner
```

Comment:
Same here, can we rename "small scale" to actual number of nodes like other tests?